remote_config/src/include/firebase/remote_config.h

```diff
@@ -115,8 +115,8 @@ enum ConfigSetting {
 };
 
 /// @brief The default cache expiration used by Fetch(), equal to 12 hours,
-/// in seconds.
-static const uint64_t kDefaultCacheExpiration = 60 * 60 * 12;
+/// in milliseconds.
+static const uint64_t kDefaultCacheExpiration = 60 * 60 * 12 * 1000;
```

Comment:
nit: For clarity and consistency, should we name this "kDefaultCacheExpirationInMilliseconds" just like "kDefaultTimeoutInMilliseconds" ?

Reply:
We could before, but I'm hesitating to change an api name now.

Reply:
Good point. I guess we will leave it as it is for now then.